src/model/ExternalFile.hpp

```diff
@@ -69,7 +69,7 @@ namespace model {
 
     static std::vector<std::string> columnSeparatorValues();
 
-    static boost::optional<ExternalFile> getExternalFile(const Model& model, const std::string& filename);
+    static boost::optional<ExternalFile> getExternalFile(const Model& model, const std::string& filename, bool copy = true);
```

Comment:
I think we want a public facing docstring to describe the method and this parameter. And maybe a more explicit name, `copyFile` or even `copyFileToWorkflowDirectory`?

First draft:

```c++
    /** This factory method will look for an existing ExternaFile with the same file name. If not it will try to create one.
      * - If copyFile is true, the file will be copied to the first entry of the WorkflowJSON's absoluteFilePaths() (or absoluteRootDir() if empty);
      * - If copyFile is false, it assumes this is an absolute path.
      * In both cases, the filename should resolve to a valid path or this will return an empty optional. */
```

Reply:
Went with `copyFile`.